src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala

```diff
@@ -59,11 +59,16 @@ object RpmPlugin extends AutoPlugin {
 
   }
 
+  private def stripSnapshot(version: String): String = {
```

Comment:
I would put the private methods at the bottom of the AutoPlugin to preserve a "general -> detailed" structure :)